### ISSUE TITLE
Parallel gif rendering

### ISSUE THREAD

**Issue by gpucce** [JuliaAnimators/Javis.jl#457]
I don't know if this is possible and even less how to do it, but wouldn't it make sense to try and parallelize the rendering of the frames to speed up the `render` function?

**Comment by Wikunia** [comment]
You mean parallelizing the ffmpeg step or the Javis side. Doing the Javis side it wouldn't work easily due to morphing where parts are computed ones but otherwise I think it should be possible.

**Comment by gpucce** [comment]
Yeah I was thinking about the ffmpeg because I believe some time ago you told me that that takes a lot of the time smth like 50% can't remember exactly.

**Comment by Wikunia** [comment]
Yeah it's time consuming. I'm not sure it's possible but if it is it would be super helpful

**Comment by Ved-Mahajan** [comment]
Can some other software be used to combine images to gifs, like [ImageMagic](https://imagemagick.org/index.php) or [libvips](libvips.org) ?
